packages/repository/test/unit/repositories/kv.repository.bridge.unit.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+
+import {
+  juggler,
+  DefaultKeyValueRepository,
+  DataObject,
+  KeyValueRepository,
+  Entity,
+} from '../../../';
+
+describe('DefaultKeyValueRepository', () => {
+  let ds: juggler.DataSource;
+  let repo: KeyValueRepository<Note>;
+
+  class Note extends Entity {
+    title?: string;
+    content?: string;
+    id: number;
+
+    constructor(data: DataObject<Note>) {
+      super(data);
+    }
+  }
+
+  beforeEach(() => {
+    ds = new juggler.DataSource({
+      name: 'db',
+      connector: 'kv-memory',
+    });
+    repo = new DefaultKeyValueRepository<Note>(Note, ds);
+  });
+
+  it('implements KeyValueRepository.set()', async () => {
+    const note1 = {title: 't1', content: 'c1'};
+    await repo.set('note1', note1);
+    const result = await repo.get('note1');
+    expect(result).to.eql(new Note(note1));
```

Comment:
Is `eql` comparing prototype/constructor? I would prefer an explicit `instanceof` check instead.

```ts
expect(result).to.be.instanceOf(Note);
```

Reply:
Yes.